jquery.kinetic.js

```diff
@@ -226,6 +224,12 @@
         if (self._useTarget(e.target, e) && self.elementFocused){
           return false;
         }
+      },
+      // prevent selection when dragging
+      selectStart: function (e){
+        if (self._useTarget(e.target, e)){
```

Comment:
But you'd need to prevent dragging if you enable selection. 

Try inverting the _useTarget? So selection is allowed on targets that don't allow dragging?